presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -147,6 +148,21 @@ public void testEndpointWithPinToCurrentRegionConfiguration()
         }
     }
 
+    @Test
+    public void testWebIdentityTokenCredentials()
+            throws Exception
+    {
+        Configuration config = new Configuration();
+        config.set(S3_IAM_ROLE, "role");
+        config.setBoolean("presto.hive.s3.web.identity.auth.enabled", true);
+        config.set("presto.hive.s3.web-identity-token-file", "/path/to/token");
```

Comment:
nit: let's use static variables defined in `S3ConfigurationUpdater` for the config names here